docs/source/guides/converting-to-openapi.rst

```diff
@@ -468,7 +468,14 @@ service.
 CORS functionality
 ------------------
 
-TODO
+When the ``smithy.api#cors`` trait is applied to a service, Smithy performs the following
+additions during the OpenAPI conversion:
+
+* Adds CORS-preflight OPTIONS requests using mock API Gateway integrations.
+* Adds CORS-specific headers to every response in the API, including ``Access-Control-Allow-Origin``,
+  ``Access-Control-Expose-Headers``, and ``Access-Control-Allow-Credentials`` where appropriate.
+* Adds static CORS response headers to API Gateway "gateway" responses.  These are added only when
```

Comment:
Can you say which gateway responses? Also, I thought it adds to all gateway responses, but just doesn't generate the DEFAULT_4XX and DEFAULT_5XX responses if they already exist.